### ISSUE TITLE
a11y for paper-submenu demo not working

### ISSUE THREAD

**Issue by ktiedt** [PolymerElements/paper-menu#44]
Trying to navigate the menu/submenu demo in the elements catalog results in a really broken keyboard test (usually focus just vanishes from the submenu when you try to navigate with any keyboard keys)

**Comment by blasten** [comment]
I was looking into this. It seems like there are two problems here  at least:

https://github.com/PolymerElements/paper-menu/blob/master/demo/index.html#L97-L104
- The paper-menu should have a `attr-for-item-title`  e.g. `attr-for-item-title="label"` so the internal paper-submenu could have `label="Topics"`
- paper-submenu should set the focus to the first item. If the first item is a paper-menu, the paper-menu should delegate focus to its first item. In this case a paper-item.

do you know where this should happen @cdata, @notwaldorf  ?

**Comment by danbeam** [comment]
what's the status on this?  it may be affecting Chrome's new PDF UI.

/cc @tjsavage @tgsergeant

**Comment by blasten** [comment]
cc @cdata  could you take another look at the PR?
